packages/miniflare/test/plugins/kv/sites.spec.ts

```diff
@@ -0,0 +1,302 @@
+import assert from "assert";
+import fs from "fs/promises";
+import path from "path";
+import anyTest, { Macro, TestFn } from "ava";
+import esbuild from "esbuild";
+import { Miniflare } from "miniflare";
+import { useTmp } from "../../test-shared";
+
+const FIXTURES_PATH = path.resolve(
+  __dirname,
+  "..",
+  "..",
+  "..",
+  "..",
+  "test",
+  "fixtures",
+  "sites"
+);
```

Comment:
nit: path.resolve will normalise `/` to `\` on Windows so you can write this as:

```suggestion
const FIXTURES_PATH = path.resolve(__dirname, "../../../../test/fixtures/sites");
```